### ISSUE TITLE
Secure internet lesson

### ISSUE THREAD

**Issue by zenrabbit** [NCEAS/oss-2017#39]
Exposure to TOR and confidential data work online.

**Comment by mbjones** [comment]
This seems like a valuable topic.   We certainly could do a quick review of good secure computing practices, such as:

- strong passwords, unique to each site, don't share
- secure communications -- Signal
- secure your phone, its the keys to the kingdom
- encryption -- whole disk, or targeted for sensitive data
- Proxy servers from random {airport, coffeeshop} 
- TOR and secure computing 

In terms of priorities, I'm not sure if we should fit it in.  Doesn't seem totally on target. Comments anyone?

**Comment by amoeba** [comment]
Doesn't feel high priority enough for a lesson but a 15 min peanut gallery slot cold work. Is that possible?

**Comment by mbjones** [comment]
Yes. Let's do that.  @amoeba would you be willing to lead a password manager demo during today's Peanut Gallery, for which we don't have anyone signed up yet?

**Comment by amoeba** [comment]
I can do it another day. I've got to step out for a personal call at 5PM today.

**Comment by mbjones** [comment]
Perfect. I forgot that I wanted to edit the code of conduct today.  So please put your name up for another day.

**Comment by amoeba** [comment]
deal!
